android/tangram/src/main/cpp/platform_android.cpp

```diff
@@ -423,7 +398,7 @@ void markerPickCallback(jobject listener, const Tangram::MarkerPickResult* marke
 
     jniEnv->CallVoidMethod(listener, onMarkerPickMID, markerPickResultObject, position[0], position[1]);
     jniEnv->DeleteGlobalRef(listener);
-
+    jniEnv->DeleteGlobalRef(tangramInstance);
```

Comment:
No it's good to have feedback on it already! 
Good call, I added the `GlobalRef` since `markerPickCallback` is called from [a lambda](https://github.com/tangrams/tangram-es/pull/1263/files/d08218364accf2d6cc30afef9adb569cc1e13a5f#diff-ca9632f59860c7cc13ebd3493d69312dR246) earlier, but the platform instance already holds a global ref on it until the disposal of the map. It can be removed but it seems safe to leave it as it is.